configuration.go

```diff
@@ -63,3 +63,16 @@ func SetRandomSeed(seed int64) {
 func SetShowStartupMessage(show bool) {
 	showStartupMessage = show
 }
+
+// SetEqualContextLineCount controls how many lines are shown around a changed diff line.
+// If set to -1 it will show full diff.
+// You should use this in the init() method of the package, which contains your tests.
+//
+// Example:
+//  init() {
+//    testza.SetEqualContextLineCount(-1) // Show all diff lines
+//    testza.SetEqualContextLineCount(3)  // Show 3 lines around every changed line
+//  }
+func SetEqualContextLineCount(lines int) {
```

Comment:
Can we find a better name for this function? It currently doesn't really describe what it's doing. Maybe something in the direction of `SetDiffContextLines` or something like that.